### PR TITLE
Component: Color

### PR DESCRIPTION
**Pull request by clrux**
This batch of work includes markup to display our library colors and their variants to aid in quick discovery of useful color combinations.

@talbs Care to review?

### CODE REVIEW

File: _src/pl/static/js/ui.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ $(document).ready(function() {
     // JS is enabled/available
     $html.removeClass('no-js');
 
-    $('a[href^="#"]').bind('click', smoothScrollLink);
```
Comment (clrux) [inline _src/pl/static/js/ui.js]: As of 1.7, the `on` method is preferred.

File: _src/pl/static/js/ui.js [hunk 2]
```diff
@@ -31,9 +31,18 @@ $(document).ready(function() {
 
     // open external links in new windows
     function newWindowLink(e) {
-        (e).preventDefault();
```
Comment (clrux) [inline _src/pl/static/js/ui.js]: `e` is a variable here, so we don't need the parens.

File: _src/pl/static/js/ui.js [hunk 3]
```diff
@@ -31,9 +31,18 @@ $(document).ready(function() {
...
+        e.preventDefault();
 
-        var $url = $(this).attr('href');
-        window.open($url);
+        window.open($(this).attr('href'));
```
Comment (clrux) [inline _src/pl/static/js/ui.js]: Little shorter.

File: _src/pl/static/js/ui.js (hunk 2 above)
Comment (talbs) [inline _src/pl/static/js/ui.js]: Cool, thanks for cleaning things up!

File: _src/pl/static/js/ui.js (hunk 3 above)
Comment (talbs) [inline _src/pl/static/js/ui.js]: Cool, thanks for cleaning things up!

File: _src/pl/static/js/ui.js (hunk 1 above)
Comment (talbs) [inline _src/pl/static/js/ui.js]: Cool, thanks for cleaning things up and helping us live in the present!

**Comment by talbs** [comment]
@clrux, here are some quick additions - https://github.com/edx/ux-pattern-library/pull/34.

**Comment by clrux** [comment]
:+1:

**Comment by clrux** [comment]
@talbs I've fixed the JS. I needed to scope the interaction to the particular `.pl-tab-wrapper` rather than all tabs and their contents. Let me know what else needs to happen with this particular piece of work so we can merge this sucker.

**Comment by talbs** [comment]
@clrux, I just pulled this branch down locally. The tabs are working, but sadly the smoothscroll from the left-hand navigation is not. Is that something that you disabled on purpose or something to fix?

**Comment by clrux** [comment]
@talbs Looks like something to fix. Didn't even notice. You know how I like those fast, instantaneous transitions and all ;) On it.

**Comment by clrux** [comment]
@talbs Fixed the smoothscroll. One more 'gain?

**Comment by talbs** [comment]
:+1:

**Pull request merged**